ivanti_service_manager/plugin.spec.yaml

```diff
@@ -734,6 +734,707 @@ types:
       description: Unread transition
       type: boolean
       required: false
+  service_request:
+    RecId:
+      title: Record ID
+      description: Record ID
+      type: string
+      required: false
+    LastModDateTime:
+      title: Last Mod Date Time
```

Comment:
Spell out Modified here. (Down below, they're using ESC, and that's spelled out in the title. Need to be consistent).

Reply:
Thanks! That's updated now :)